doc/model/fluid.rst

```diff
@@ -0,0 +1,176 @@
+=====
+Fluid
+=====
+
+.. note::
+
+   This model document has been updated to be *MIRGE-Com* specific, and to include multi-component mixtures
+   with chemical reactions.
+
+.. important::
+
+   *MIRGE-Com* main repo branch currently provides fluid simulation operators and utilities for inviscid
+   flow (i.e. the :ref:`Euler equations<Euler-eqns>`).  Viscous flow simulation capabilities are forthcoming.
+
+.. raw:: latex
+
+    \def\RE{\operatorname{RE}}
+    \def\PR{\operatorname{PR}}
+
+.. raw:: html
+
+    \(
+    \def\RE{\operatorname{RE}}
+    \def\PR{\operatorname{PR}}
+    \)
+
+.. _NS-eqns:
+
+*MIRGE-Com* provides capabilities for solving the compressible Navier-Stokes equations for viscous flows and
+the :ref:`Euler equations<Euler-eqns>` equations for inviscid flows of reactive fluid mixtures. *MIRGE-Com*
+supports reactive fluid mixtures with a number of mixture species = $N_s$ on unstructured meshes and
+:ref:`discretizes the equations in a Discontinuous Galerkin setting<disc-strat>`.
+
+The formulation presented here follows from [Ihme_2014]_ and [Cook_2009]_. The governing equations, written in conservative form, are summarized as follows:
+
+.. math::
+    \partial_{t}{\rho} + \partial_{j}{\rho v_j} &= S_\rho \\
+    \partial_{t}(\rho{E}) + \partial_j\left(\left\{\rho E + p\right\}v_j + q_j - \tau_{jk}v_k\right) &= S_{\rho E} \\
+    \partial_{t}({\rho}{v_i}) + \partial_j\left(\rho v_i v_j + p\delta_{ij} - \tau_{ij}\right) &= S_{\rho v_i} \\
+    \partial_{t}(\rho{Y}_{\alpha}) + \partial_j\left(\rho{Y}_{\alpha}v_j + (\mathbf{J}_{\alpha})_j\right) &= S_{\alpha},
+
+with fluid density $\rho$, velocity components $v_i$, momentum density components $\rho v_i$, total energy $\rho E$,
+and vector of species mass fractions ${Y}_{\alpha}$. The :ref:`thermodynamic pressure<eos-and-matprop>` of the fluid
+is $p$.  ${\tau_{ij}}$ are the components of the :ref:`viscous stress tensor<viscous-stress-tensor>`, $q_i$ are the
+components of the total :ref:`heat flux<heat-flux>` vector, and the components of the
+species :ref:`diffusive flux<diffusive-flux>` vector are $(\mathbf{J}_{\alpha})_i$. Mixtures have $N_s$ components
+with $1 \le \alpha \le N_s$. Unless otherwise noted, repeated indices imply summation.
+
+The equations can be recast in a more compact form:
+
+.. math::
+
+    \partial_t{\mathbf{Q}} + \partial_j{\mathbf{F}^{I}_j} = \partial_j{\mathbf{F}^{V}_j} + \mathbf{S},
```

Comment:
`F(Q)` instead of just `F`?

Reply:
I waffled around with this.  I think it looks better without.  I think its dependence on Q is clear - but up to you guys.

Reply:
I'm pretty sure @thomasgibson voted for *with* explicit Q, grad(Q) dependence and added it a suggestion in his last set of comments.